public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -111,68 +112,70 @@ export default class FindingDetailsFlyout extends Component<
         <div key={key}>
           <EuiAccordion
             id={`${key}`}
+            className="euiAccordionForm"
+            buttonClassName="euiAccordionForm__button"
             buttonContent={
               <div data-test-subj={'finding-details-flyout-rule-accordion-button'}>
-                <EuiText size={'s'}>{fullRule.title}</EuiText>
-                <EuiText size={'s'} color={'subdued'}>
-                  Severity: {severity}
-                </EuiText>
+                <EuiTitle size={'s'}>
+                  <h3>Rule details</h3>
+                </EuiTitle>
```

Comment:
We want to still keep the title here along with severity since there can be multiple rules.

Reply:
Yes, you are right.

Reply:
@jovancacvetkovic Do you have an updated screenshot of how this look like? Thanks.

Reply:
@kamingleung added a few screenshots.